pkgs/applications/audio/faust/faust2.nix

```diff
@@ -194,8 +196,8 @@ let
         # export parts of the build environment
         for script in "$out"/bin/*; do
           wrapProgram "$script" \
-            --set FAUSTLIB "${faust}/lib/faust" \
-            --set FAUST_LIB_PATH "${faust}/lib/faust" \
+            --set FAUSTLIB "${faust}/share/faust" \
```

Comment:
I just built your PR ( you need `stdenv.lib.getVersion`) and saw it still keeps the shared libraries in `lib` so that is fine. I suppose `FAUSTLIB` affects the `.lib` files.



and saw a couple of things:
-